packages/framework-provider-aws/test/library/events-adapter.test.ts

```diff
@@ -130,6 +130,33 @@ describe('the events-adapter', () => {
         })
       )
     })
+
+    it('publishes the eventEnvelopes passed via parameter in batch of 25 elements', async () => {
+      const config = new BoosterConfig('test')
+      config.appName = 'test-app'
+      const event: EventEnvelope = {
+        version: 1,
+        entityID: 'id',
+        kind: 'event',
+        value: {
+          id: 'id',
+        },
+        typeName: 'EventName',
+        entityTypeName: 'EntityName',
+        requestID: 'requestID',
+        createdAt: 'once',
+      }
+      const eventEnvelopes = Array.from({ length: 51 }, () => event)
+
+      const fakeBatchWrite = fake.returns({
+        promise: fake.resolves(''),
+      })
+      const fakeDynamo: DocumentClient = { batchWrite: fakeBatchWrite } as any
+
+      await Library.storeEvents(fakeDynamo, eventEnvelopes, config, fakeLogger)
+
+      expect(fakeBatchWrite).to.be.calledThrice
```

Comment:
Would it make sense to check that each call contains the right number of items and that they're in the expected order?

Reply:
Agree here too. It would be really dangerous that, because of rounding problems, we forget to persist some events